instagram-app/.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
Not really much to review since we solved issues you were having in stand up! Enjoy your sleep :D